### ISSUE TITLE
Error in foamMonitor

### ISSUE THREAD

**Issue by cgramesp** [mrklein/openfoam-os-x#31]
It seems that foamMonitor ist not working under Mac OS. Code to reproduce:

```
   cd $FOAM_TUTORIALS/incompressible/simpleFoam/pitzDaily/
   ****add "#includeFunc residuals" to functions in controlDict
   blockMesh
   simpleFoam > log &
   foamMonitor -l postProcessing/residuals/0/residuals.dat 
```

Error:

```
stat: illegal option -- -
usage: stat [-FlLnqrsx] [-f format] [-t timefmt] [file ...]
/Users/USERNAME/OpenFOAM/OpenFOAM-4.x/bin/foamMonitor: line 174: test: : integer expression expected
/Users/USERNAME/OpenFOAM/OpenFOAM-4.x/bin/foamMonitor: line 179:  7922 Killed: 9               $GNUPLOT $GPFILE
```

Works fine on Linux. I suppose that my Gnuplot installation is fine. Could you provide any solution?

**Comment by mrklein** [comment]
It really does not work. In the case, you have provided, there were two things:
1. macOS `stat` command uses different flags.
2. My `gnuplot` was built without X11 terminal, so after I have fixed problem with flags, I have got problem with terminal name. If your `gnuplot` has this terminal, maybe you can keep line 60 as it is.

To fix this, you need to edit `$WM_PROJECT_DIR/bin/foamMonitor`:
1. Line 60, change `x11` to `wxt`.
2. Line 172, change `--format=%Y` to `-f %m`.

I have attached zipped patch file, which do this. Copy it to `$WM_PROJECT_DIR`, unzip, and run `git apply foamMonitor.patch` there.

[foamMonitor.patch.zip](https://github.com/mrklein/openfoam-os-x/files/769641/foamMonitor.patch.zip)

**Comment by cgramesp** [comment]
Thanks for your fast answer!

The patch fails on line 57 in `$WM_PROJECT_DIR/bin/foamMonitor`:

```
error: patch failed: bin/foamMonitor:57
error: bin/foamMonitor: patch does not apply
```

**Comment by cgramesp** [comment]
Changing those two lines results in the following error while executing foamMonitor:

```
set term wxt 1 font "helvetica,17" linewidth 1.5 persist noraise
         ^
"/var/folders/cf/j086kf2n5rj2d6qzzh1cf_mc0000gn/T/tmp.5BanbTxU", line 1: unknown or ambiguous terminal type; type just 'set terminal' for a list
```

**Comment by mrklein** [comment]
Guess, you are trying to apply patch after editing file. It is either you edit file, or apply patch. I have corrected wording in my previous comment.

Concerning terminal name: I have `gnuplot` installed with `--with-wxmac` option, so I do not have `x11` terminal and do have `wxt`. In your case it can be otherwise. So either you can keep `x11`, or see the list of available terminals in `gnuplot` and change line 60 accordingly.

**Comment by mrklein** [comment]
Resolved by 58e74a32ac72c9cee1ec1e432af53ef374aca4fe

**Comment by mrklein** [comment]
And stats flag was not corrected in 58e74a3
